lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -27,9 +27,15 @@ class TomeeInstance < TomcatInstance
       def compile
         download(@version, @uri) { |file| expand file }
         link_to(@application.root.children, root)
-        unless ear?
-          @droplet.additional_libraries << tomcat_datasource_jar if tomcat_datasource_jar.exist?
+        if ear?
+          # if there is a drivers subfolder in the ear app, then link the libraries over to tomee/lib folder
+          if drivers?
+            link_to((@application.root + 'drivers').children, web_inf_lib)
```

Comment:
What do you think about renaming `web_inf_lib` method to something else for example `lib_folder` or something else?
Now it is not just `WEB-INF/lib` but it might be `EAR/drivers`